app/graphql/types/rule.rb

```diff
@@ -52,12 +45,12 @@ def failed_count
 
     private
 
-    def system_id(args)
-      args[:system_id] || context[:parent_system_id]
+    def system_id
+      context[:parent_system_id]
     end
 
-    def profile_id(args)
-      args[:profile_id] || context[:parent_profile_id][object.id]
+    def profile_id
+      context[:parent_profile_id][object.id]
```

Comment:
This could result with:
```
undefined method `[]' for nil:NilClass (NoMethodError)
```
when calling compliant for a Rule outside of a profile.